Parse/src/test/java/com/parse/ParseObjectTest.java

```diff
@@ -237,7 +238,16 @@ public void testGetUnavailable() {
     ParseObject.State state = mock(ParseObject.State.class);
     when(state.className()).thenReturn("TestObject");
     when(state.isComplete()).thenReturn(false);
+    ParseObject object = ParseObject.from(state);
+    object.get("foo");
```

Comment:
No, this checks that the exception is raised (see the @Test annotation). Actually I have not changed this method, I have added testGetAvailableIfKeySafe() a few lines below, that tests safeKeys() .